Core/src/org/sleuthkit/autopsy/textextractors/SqliteTextExtractor.java

```diff
@@ -204,7 +204,7 @@ private void fillBuffer(String val) {
          */
         @Override
         public int read(char[] cbuf, int off, int len) throws IOException {
-            buf = cbuf;
+            buf = cbuf.clone();
```

Comment:
Is the caller expecting this method to fill the cbuf array, but we are now filling a clone of it? Frankly, I don't understand why buf is a instance variable anyway - it is only referenced in this single read method...

Reply:
Apparently yes we need buf to equal cbuf not a copy of it. So I will undo this change, and leave it as a codacy error.